src/Components/Facility/AddInventoryForm.tsx

```diff
@@ -211,9 +211,16 @@ export const AddInventoryForm = (props: any) => {
         pathParams: { facilityId },
         onResponse: ({ res, data }) => {
           if (res?.ok && data) {
-            Notification.Success({
-              msg: "Inventory created successfully",
-            });
+            if (data.is_incoming) {
+              Notification.Success({
+                msg: "Inventory created successfully",
+              });
+            } else if (!data.is_incoming) {
```

Comment:
```suggestion
            } else {
```

`else if (!data.is_incoming)`

Since this is an else block of `if (data.is_incoming)` , it will by default run only when `!data.is_incoming` so we do not need this additional if check.